src/components/Modals/Metadata/Metadata.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Box, Button, Image, Text } from 'grommet'
+import { Close } from 'grommet-icons'
+import styled from 'styled-components'
+import data from './mockData'
+import Satellite from 'images/satellite_map.png'
+import ConditionalLink from './components/ConditionalLink'
+
+const Uppercase = styled(Text)`
+  text-transform: uppercase;
+`
+
+export default function Metadata() {
+  return (
+    <Box
+      border={{ color: 'kelp' }}
+      elevation='small'
+      gap='xsmall'
+      height='32rem'
+      pad='medium'
+      width='27rem'
+    >
+      <Box direction='row' justify='between'>
+        <Text color='kelp'>Subject Metadata</Text>
+        <Button
+          gap='xsmall'
+          icon={<Close color='black' size='small' />}
+          label={<Uppercase color='kelp' size='small'>Close</Uppercase>}
+          onClick={() => console.log('Close the modal')}
+          plain
+          reverse
+        />
+      </Box>
+      <Box
+        background='sand'
+        border={{ color: 'kelp' }}
+        height={{ max: '16rem' }}
+      >
+        <Image fit='contain' src={Satellite} />
+      </Box>
+      <Box gap='xxsmall' overflow={{ vertical: 'auto' }}>
+        {Object.entries(data).map(datum => {
+          const [key, value] = datum
+          return (
+            <Box direction='row' gap='xsmall'>
+              <Box align='end' basis='1/3'>
+                <Uppercase color='kelp'>{key}</Uppercase>
+              </Box>
+              <Box basis='2/3'>
+                <ConditionalLink color='kelp' text={value} />
+              </Box>
+            </Box>
```

Comment:
I'd suggest using a HTML table for screen readability for this data. The flexbox divs here don't really communicate the relationship between the keys and values here, but a table can semantically be configured to do so. We use the Grommet `DataTable` with the classifier metadata modal:

https://github.com/zooniverse/front-end-monorepo/blob/be8959d36c95d210e94cebaa01307283a215c2da/packages/lib-classifier/src/components/Classifier/components/MetaTools/components/Metadata/components/MetadataModal/MetadataModal.js#L75-L87